src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayPrice = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days < 3) {
+    return (days * dayPrice);
```

Comment:
For expression `days * dayPrice`, you should create a new var, as well
For example: `const totalAmount = days * dayPrice`